system/core/editor.php

```diff
@@ -149,7 +149,16 @@ function loadEditor($html) {
 					this.isFocusSet = true;
 				}
 
-				// initiate the "image_upload" plugin
+                wym._iframe.contentDocument.addEventListener('paste', (event) => {
```

Comment:
There's again whitespace issues here (you're inserting spaces, while it should be tabs).

What editor are you using? We could maybe add a `.editorconfig` file to the project if your editor supports that (though I'm not sure if the indentation style is sufficiently consistent throughout the project...).

Reply:
fixed